components/dashboard/src/service/service.tsx

```diff
@@ -191,7 +191,10 @@ export class IDEFrontendService implements IDEFrontendDashboardService.IServer {
         let redirect = false;
         try {
             const desktopLink = new URL(url);
-            redirect = desktopLink.protocol !== "http:" && desktopLink.protocol !== "https:";
+            // allow to redirect only for whitelisted trusted protocols
+            // security: IDE-69
```

Comment:
Drop the security line. One of the first things I'll do against an app is decompile/get the source and search for keywords like "hack", "security" etc